app/views/foreman_snapshot_management/hosts/_snapshots_tab.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if @host.provider == 'VMware' %>
+<% if @host.compute_resource && @host.compute_resource.capable?(:snapshots) && authorized_for(:controller => 'foreman_snapshot_management/snapshots', :action => :index) %>
```

Comment:
`capable?` has been introduced in theforeman/foreman@3cf9d65f523930a425c49a186fe491d70c5483ca which is, sadly, only in `develop` (and the 1.16 tree) by now.
Engine says, we require foremen >= 1.14. Bumping that to a stable version would be ok for me.

Reply:
We can simply fall back to `.capabilities.include?`. Then we should be covered for 1.14 + 
Thanks for spotting this.